service-rpc/src/main/thrift/cluster.thrift

```diff
@@ -350,6 +350,8 @@ service TSDataService extends RaftService {
 
   list<binary> getAggrResult(1:GetAggrResultRequest request)
 
+  map<string, bool> isMeasurementsRegistered(1: Node header, 2: list<string> measurements)
```

Comment:
A suggestion about the interface design: I think the return type can be a list<bool> with the length of `measurements` because the sender clearly know what series it queries and the receiver need not send them back.

Another thing, I think "measurement" is not proper here, better just call them timeseries.

Reply:
I will change the interface, specifically, the return, to be a list f string, `List<String>,` which are  unregistered series.